cmd/contour/serve.go

```diff
@@ -87,6 +89,8 @@ func registerServe(app *kingpin.Application) (*kingpin.CmdClause, *serveContext)
 
 	serve.Flag("http-address", "Address the metrics http endpoint will bind to.").StringVar(&ctx.metricsAddr)
 	serve.Flag("http-port", "Port the metrics http endpoint will bind to.").IntVar(&ctx.metricsPort)
+	serve.Flag("health-address", "Address the health http endpoint will bind to.").StringVar(&ctx.healthAddr)
+	serve.Flag("health-port", "Port the health http endpoint will bind to.").IntVar(&ctx.healthPort)
```

Comment:
I don't know why HTTP wasn't capitalized originally, can you please capitalize it for both flag sets?